article/helpers.py

```diff
@@ -86,6 +84,7 @@ def remaining_reading_time_in_group(self, group_name):
         """ Return the remaining reading time in minutes
             for unread articles in the group
         """
+
         read_articles_uuids = self.read_articles_keys_in_group(group_name)
```

Comment:
It's not relevant to your code, but I'd change this to `read_articles_uuids_in_group` for consistency